sentry-spring/src/main/java/io/sentry/spring/reactive/SentryReactiveWebHelper.java

```diff
@@ -0,0 +1,23 @@
+package io.sentry.spring.reactive;
+
+import io.sentry.IHub;
+import io.sentry.ILogger;
+import io.sentry.SentryLevel;
+import io.sentry.SystemOutLogger;
+import java.util.function.Consumer;
+import org.springframework.web.server.ServerWebExchange;
+
+public final class SentryReactiveWebHelper {
+  static final String REQUEST_HUB_ATTR_NAME = "SentryReactiveWebHelper.REQUEST_HUB_ATTR_NAME";
+
+  private static final ILogger LOGGER = new SystemOutLogger();
```

Comment:
can you get the instance from `options.getLogger()`? the SDK already has its own logger